examples/assets_pandas_type_metadata/assets_pandas_type_metadata_tests/test_defs.py

```diff
@@ -0,0 +1,5 @@
+from assets_pandas_type_metadata import defs
+
+
+def test_defs_can_load():
+    defs.get_repository_def().load_all_definitions()
```

Comment:
`load_all_definitions` doesn't really do much in this case. In fact it's not clear we should recommend its use at all since AFAIR it mainly used to force loading definitions in a rarely used format where you can pass in a dictionary of callables into a repository.

Seems like we need to add a get_all_assets accessor to Definitions?

Reply:
yea `get_all_assets` would be great but iirc it's not available on repository either tho